src/main.rs

```diff
@@ -174,6 +174,14 @@ fn main() {
         }
     }
 
+    // Optionally add a newline to the end of the file.
+    // This is placed after the check for if the file is smaller. It's OK if the
+    // file grows when adding a newline, since the user explicitly wanted that.
+    let newline_flag = args.value_of(KEYS[Key::OutputNewline]).unwrap();
```

Comment:
I think we can use `cleaning_opt.output_newline` here. No need to parse it again.

Reply:
Yes, that works. I originally tried doing this for `WriteOptions` but was told it's not a member.